frontend/logout.ts

```diff
@@ -67,7 +67,7 @@ async function logoutSession(session) {
     });
 
     promise.catch(error => {
-        console.log(`Failed to logout chatrix session. deviceId: ${session.deviceId}`, error);
+        console.error(`Failed to logout chatrix session. deviceId: ${session.deviceId}`);
```

Comment:
Was the 2nd arg mistakenly removed?

Reply:
I removed it deliberately since it will be logged by the exterior catch, this catch is so that it's possible to know for which session logout failed. Maybe there would be a cleaner way to do this.

Reply:
Gotcha!